packages/ocaml-variants/ocaml-variants.4.10.2+32bit/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis:
+  "Official release 4.10.2, compiled in 32-bit mode for 64-bit Linux and OS X hosts"
+maintainer: "platform@lists.ocaml.org"
+authors: "Xavier Leroy and many contributors"
+homepage: "https://ocaml.org"
+bug-reports: "https://github.com/ocaml/ocaml/issues"
+dev-repo: "git://github.com/ocaml/ocaml"
+depends: [
+  "ocaml" {= "4.10.2" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+]
+depexts: [
+  ["gcc-multilib" "g++-multilib"] {os-family = "debian"}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "CC=gcc -m32"
+    "AS=as --32"
+    "ASPP=gcc -m32 -c"
+    "--host" "i386-linux"
+    "PARTIALLD=ld -r -melf_i386"
+  ] {os = "linux"}
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "CC=gcc -Wl,-read_only_relocs,suppress -arch i386 -m32"
+    "AS=as -arch i386"
+    "ASPP=gcc -arch i386 -m32 -c"
+    "--host" "i386-apple-darwin13.2.0"
```

Comment:
I'm a little suspicious about this line. Why's it setting i386-apple-darwin13.2.0 for OpenBSD and FreeBSD as well? And I believe 32-bit is no longer supported on macOS anyway.  Not a blocker to this PR, as I expect this is propagated from earlier versions of this file.

Reply:
This was indeed propagated from the 4.10.1 file... The new ocaml-option scheme in 4.12 is really nice from this point of view.